pantheon-bundle/frontend/src/app/searchBeta.tsx

```diff
@@ -0,0 +1,570 @@
+            keyWord={this.state.inputValue}
+            repositoriesSelected={this.state.repositoriesSelected}
+            productsSelected={this.state.productsSelected}
+            userAuthenticated={this.props.userAuthenticated}
+            filters={this.state.filters}
+          />
+
+        </ExpandableSection>
+        <br />
+        <ExpandableSection toggleText="Assemblies" className="pf-c-title" isActive={true} isExpanded={assembliesIsExpanded} onToggle={this.onAssembliesToggle}>
+          <SearchResults
+            contentType="assembly"
+            keyWord={this.state.inputValue}
+            repositoriesSelected={this.state.repositoriesSelected}
+            productsSelected={this.state.productsSelected}
+            userAuthenticated={this.props.userAuthenticated}
+            filters={this.state.filters}
+          />
+
+        </ExpandableSection>
+      </React.Fragment>
+    );
+
+    const statusMenuItems = [
+      <SelectOption key="statusDraft" value="draft" label= "Draft" />,
+      <SelectOption key="statusPublished" value="released" label="Published" />
+    ];
+
+    const contentTypeMenuItems = [
+      <SelectOption key="ctypeConcept" value="CONCEPT" label="Concept" />,
+      <SelectOption key="ctypeProcedure" value="PROCEDURE" label="Procedure" />,
+      <SelectOption key="ctypeReference" value="REFERENCE" label="Reference" />
+    ];
+
+    const toggleGroupItems = (
+      <React.Fragment>
+        <ToolbarItem>
+          <Button variant="tertiary" aria-expanded={isExpanded} onClick={this.onClick} icon={<FilterIcon />} />
+        </ToolbarItem>
+        <ToolbarItem>
+          <InputGroup>
+            <SearchInput
+              name="textInput"
+              id="textInput"
+              placeholder="Find by name"
+              type="search"
+              aria-label="search input"
+              onChange={this.onInputChange}
+              onClear={this.onInputClear}
+              value={inputValue}
+            />
+            <Button variant={ButtonVariant.control} aria-label="search button for search input">
+              <SearchIcon />
+            </Button>
+          </InputGroup>
+        </ToolbarItem>
+        <ToolbarGroup variant="filter-group">
+          <ToolbarFilter
+            chips={filters.status}
+            // deleteChip={this.onDelete}
+            deleteChipGroup={this.onDeleteGroup}
+            categoryName="Status"
+          >
+            <Select
+              variant={SelectVariant.checkbox}
+              aria-label="Status"
+              onToggle={this.onStatusToggle}
+              onSelect={this.onStatusSelect}
+              selections={filters.status}
+              isOpen={statusIsExpanded}
+              placeholderText="Status"
+            >
+              {statusMenuItems}
+            </Select>
+          </ToolbarFilter>
+          {/* <ToolbarFilter chips={filters.ctype} deleteChip={this.onDelete} categoryName="Content Type"> */}
```

Comment:
Is this duplicate code? Looks the same starting on line 249

Reply:
This is not duplicated code, line 248 enables "deleteChip" function while line 249 doesn't. Ideally, we want to have "deleteChip" enabled. this work effort is captured in CCS-3975

Reply:
Ah got it, thanks for clarifying